code/src/Album.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import data from './data.json';
+import 'components/Album.css';
+
+
+
+/*console.log(data);*/
+/*console.log(data.albums.items[0])*/
+
+
+export const Album = (props) => {
+    return (
+        <div className="album">
```

Comment:
Looks like you're editor is set up to do 4 spaces for indentation (should be 2). See [this question on stack overflow](https://stackoverflow.com/c/technigo/questions/253) for a bunch of things to check.